### PR TITLE
Fix returning cursor

### PR DESCRIPTION
**Pull request by kalashnikovisme**
Fixed this issue https://github.com/AndrewRedican/react-json-editor-ajrm/issues/88

### CODE REVIEW

**Comment by andycloke** [comment]
@kalashnikovisme given this repo is not maintained and this is unlikely to be meregd, is there any chance you could build the files in your version so people can install it directly from your repo?

File: src/index.js [hunk 1]
```diff
@@ -502,7 +504,7 @@ class JSONInput extends Component {
             lines      : data.lines,
             error      : data.error
         });
-        let cursorPosition = this.getCursorPosition(data.error) + cursorOffset;
+        cursorPosition = cursorPosition == 0 ? this.getCursorPosition(data.error) + cursorOffset : cursorPosition;
```
Comment (AndrewRedican) [inline src/index.js:507]: Can we use strict comparison here?

**Comment by AndrewRedican** [comment]
@kalashnikovisme happy to merge this in. There was a lot of catching up to do. Do you mind resolving the conflict, I'll be happy to merge it then.

**Comment by kalashnikovisme** [comment]
@AndrewRedican @andycloke found today some new bugs with this solution. Will fix them in a few days.
Don't close this PR until we finish work, please.

Thnx!

**Comment by hirokib** [comment]
@kalashnikovisme how is the bug work coming along? Just came across this thread and would love a solution to this.

**Comment by kalashnikovisme** [comment]
@hirokib unfortunately we decided to not use this package for now. Selected another packaged. We hope that we will have resources to fix bugs at this solution properly later

**Comment by collinscangarella** [comment]
@kalashnikovisme Which package did you end up using? We're reconsidering whether or not we want to use this package due to this bug and some other usability issues.

**Comment by kalashnikovisme** [comment]
@collinscangarella we've decided to use this solution https://github.com/josdejong/jsoneditor with adding this to react https://github.com/josdejong/jsoneditor/tree/develop/examples/react_advanced_demo

It works fine through it's not react package.

**Pull request closed without merging**